app/src/main/res/values/strings.xml

```diff
@@ -12,20 +12,19 @@
     <string name="page_2_toolbar_title">Example Request</string>
     <string name="fragment_example_username">Username</string>
     <string name="fragment_example_login">Login</string>
-    <string name="vlogin_empty_edit_text_messaje">Los campos de correo electronico y contraseña son requeridos</string>
-    <string name="vlogin_empty_email">El campo de correo electronico es requerido</string>
-    <string name="vlogin_empty_password">El campo de contraseña es requerido</string>
-    <string name="vlogin_invalid_email">Correo electronico invalido</string>
-    <string name="vlogin_email">example@domain.com</string>
-    <string name="vlogin_preferences_name">ar.com.wolox.android.example.ui.login</string>
-
-    <string name="vlogin_username">usuario</string>
-    <string name="vlogin_password">contraseña</string>
-    <string name="vlogin_button_login">LOG IN</string>
-    <string name="vlogin_button_signup">SIGN UP</string>
-    <string name="vlogin_terms_and_conditions">By joing you agree to our <u>Terms &amp; Conditions</u></string>
-    <string name="vhome__activity">HOME</string>
+    <string name="login_empty_edit_text_messaje">Los campos de correo electronico y contraseña son requeridos</string>
+    <string name="login_empty_email">El campo de correo electronico es requerido</string>
+    <string name="login_empty_password">El campo de contraseña es requerido</string>
+    <string name="login_invalid_email">Correo electronico invalido</string>
+    <string name="login_email">example@domain.com</string>
+    <string name="login_preferences_name">ar.com.wolox.android.example.ui.login</string>
+    <string name="login_username">usuario</string>
+    <string name="login_password">contraseña</string>
+    <string name="login_button_login">LOG IN</string>
+    <string name="login_button_signup">SIGN UP</string>
+    <string name="login_terms_and_conditions">By joing you agree to our <u>Terms &amp; Conditions</u></string>
+    <string name="home__activity">HOME</string>
     <string name="password_required">Password is Required</string>
-
+    <string name="vRootUsername">username</string>
```

Comment:
This should not be saved as a string resource. It should be a constant value inside the class that implement it.